src/main/twirl/uk/gov/hmrc/hmrcfrontend/views/helpers/HmrcLayout.scala.html

```diff
@@ -53,6 +53,8 @@
 
 @require(!(backLinkUrl.isDefined && backLink.isDefined), "Use backLink or backLinkUrl, but not both")
 
+@noLayoutWrapper = @{ identity[Html] _ }
```

Comment:
given this a name for clarity, but if we think the test shows intent enough then could just inline `identity`

Reply:
I like it having a name 🙌